source/server/http/admin.cc

```diff
@@ -31,8 +31,17 @@
 #include "common/upstream/host_utility.h"
 
 #include "fmt/format.h"
+#include "rapidjson/document.h"
```

Comment:
I would prefer that we don't like rapidjson out into the rest of the code, but it's OK for now. Can you please add above the includes:

"TODO(mattklein123): Switch to JSON interface methods and remove rapidjson dependency"